app/controllers/Checkout.scala

```diff
@@ -1,46 +1,41 @@
 package controllers
 
 import actions.CommonActions._
-import com.gu.identity.play.IdUser
+import com.gu.identity.play.{AuthenticatedIdUser, IdUser}
 import com.typesafe.scalalogging.LazyLogging
 import configuration.Config.Identity.webAppProfileUrl
 import forms.{FinishAccountForm, SubscriptionsForm}
 import model.SubscriptionData
 import play.api.data.Form
 import play.api.libs.json._
-import play.api.mvc
 import play.api.mvc._
+import services.AuthenticationService.authenticatedUserFor
 import services.CheckoutService.CheckoutResult
 import services._
-import utils.TestUsers
 import utils.TestUsers.{NameEnteredInForm, PreSigninTestCookie}
 import views.html.{checkout => view}
-
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
-import AuthenticationService.authenticatedUserFor
 
 object Checkout extends Controller with LazyLogging {
 
+  def fillForm(form: Form[SubscriptionData], authUserOpt: Option[AuthenticatedIdUser]): Future[Form[SubscriptionData]] = {
```

Comment:
I agree with moving this one out of `renderCheckout`, but I would also mark it as private